full_name.rb

```diff
@@ -0,0 +1,16 @@
+def full_name
```

Comment:
Nice job. It was a good idea to separate out the code into the ask method so that you don't write duplicate code.